packages/blocks/src/api/parser.js

```diff
@@ -191,15 +191,13 @@ export function matcherFromSource( sourceConfig ) {
 
 			return matcher;
 		case 'html':
-			return html( sourceConfig.selector );
+			return html( sourceConfig.selector, sourceConfig.multiline );
```

Comment:
It's not obvious what purpose `multiline` serves when considering `html` in isolation outside the RichText use-case. We're kinda muddying the distinction between these sources.